build/protoc/genproto.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+gen_proto() {
+  output_dir=$1
```

Comment:
Probably really dockerdumb question but I'm having a hard time tracing where we are actually passing the output_dir as an argument to this script...I see we're mounting the /client and /docs directories in the `proto_command` defined in the Makefile but is there somewhere else we specify an argument to this script?

Reply:
Good question, this is somewhat confusing bash...I think I've written my yearly quota of bash so I'll need to pick another language if this script gets any more complicated.

We don't pass the `output_dir` to the script itself, this is passed in two places within the script:
* `gen_proto /output` in `update_proto`
* `gen_proto /staging` in `check_proto`

Reply:
Oh okay I see, thanks!